android/app/src/main/java/com/on/turip/di/DeviceModule.kt

```diff
@@ -0,0 +1,17 @@
+package com.on.turip.di
+
+import com.on.turip.platform.device.AndroidDeviceInfoProvider
+import com.on.turip.platform.device.DeviceInfoProvider
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class DeviceModule {
+    @Binds
+    @Singleton
+    abstract fun provideDeviceInfoProvider(impl: AndroidDeviceInfoProvider): DeviceInfoProvider
+}
```

Comment:
```suggestion
    abstract fun provideDeviceInfoProvider(androidDeviceInfoProvider: AndroidDeviceInfoProvider): DeviceInfoProvider
}
```
c: impl 대신에 이렇게 써왔던 거 같은데 바꾸어 보면 어떨까유?

Reply:
통일성 좋네요!

[refactor: DeviceModule 함수 파라미터명 수정](https://github.com/woowacourse-teams/2025-Turip/pull/498/commits/2fb98ed9c93c42e345e3de9ac6b1d0e8f8abae2f)